server/admin/reload.js

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2015 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
same functionality as before. just moved out of `server/index.js` and into its own file.